src/main/java/com/flightstats/hub/spoke/SpokeManager.java

```diff
@@ -383,4 +391,13 @@ public void run() {
         return countDownLatch.await(60, TimeUnit.SECONDS);
     }
 
+    /**
+     * @return the localhost's server
+     */
+    private Collection<String> getLocalServer() {
+        List<String> server = new ArrayList<>();
+        server.add(localHostProperties.getHost(false));
+        return server;
```

Comment:
I think this could be done in one line with `return Arrays.asList(localHostProperties.getHost(false));`